examples/fluent-bit/ecs-log-deletion/README.md

```diff
@@ -0,0 +1,135 @@
+* The volume mount for Fluent Bit must not be read-only
+* Familiar with [exec input](https://docs.fluentbit.io/manual/pipeline/inputs/exec) of Fluent Bit
+
+You can run a command that deletes log files by adding an exec input to the configuration. Here is the example:
+
+```
+[INPUT]
+    Name          exec
+    Tag           delete-old-files
+    Command       find /var/log/  -type f -mmin +10080 -execdir  rm -f -- '{}' ';'
+    Interval_Sec  600
+    Interval_NSec 0
+    Buf_Size      8mb
+    Oneshot       false
+
+[OUTPUT]
+    Name          null
+    Match         delete-old-files
+```
+Here is a breakdown of the individual components of the command:
+* ```find /var/log/```: This is the ```find``` command, which searches for files in the directory ```/var/log/```.
+* ```-type f```: This option tells find to search only for regular files, not directories or other types of files.
+* ```-mmin +10080```: This option tells find to search for files that are older than 7 days (10080 minutes).
+* ```-execdir rm -f -- '{}'';'```: This is the action to take on the files that match the search criteria. The ```-execdir``` option runs the ```rm``` command in the directory where each file is found. The ```-f``` option to ```rm``` tells it to force the deletion without prompting for confirmation. The ```--```is a separator that indicates the end of the command options and the start of the filenames, and ```{}``` is a placeholder that gets replaced with the name of each file found by find. Finally, the ```';'``` marks the end of the command that should be executed for each file.
+
+### 2. Run deletion command with cron
+
+##### Prerequisite:
+* Familiar with [cron](https://en.wikipedia.org/wiki/Cron)
+* Docker should be installed on your system
+
+**1. Create a cron job file and write the command below inside the file**
+```
+0 0 * * * find /var/log/ -type f -mmin +10080 -execdir rm -f -- '{}' ';'
+```
+This is a cron job that runs every day at midnight (```0 0 * * *```). It uses the ```find``` command to locate files in the directory ```/var/log/``` that are older than 7 days (10080 minutes), and then deletes them using the ```rm -f``` command.
+* ```0 0 * * *```: This is the cron schedule expression. It specifies that the job should run at 12:00 AM every day.
+* Note: In the cron job file called "cron", an empty line is required at the end of the file for a valid cron file.
+
+**2. Build image with cron**
+* Updating the packages and installing cron
+```
+RUN apt-get update && apt-get -y install cron -y
+```
+* Copy cron file to the cron.d directory
+```
+COPY cron /etc/cron.d/cron
+```
+* Give execution rights on the cron job
+```
+RUN chmod 0644 /etc/cron.d/cron
+```
+* Apply cron job
+```
+RUN crontab /etc/cron.d/cron
+```
+* Create the log file to be able to run tail
+```
+RUN touch /var/log/cron.log
+```
+* Run the command on container startup
+```
+CMD cron && tail -f /var/log/cron.log
+```
```

Comment:
what does this do? Why do we need to tail the cron log? I think this is optional and not needed. I